data/outreachy.yml

```diff
@@ -0,0 +1,138 @@
+        description: >
+          Usually, the output messages from the compiler are a bit more difficult to read for a machine
+          and hence it's more time consuming to find the warnings, errors, etc. and their origin. By producing
+          a structured output for compiler messages, other tools can more easily interoperate with them and
+          provide tooling on top of the messages.
+        mentee: "Muskan Garg"
+        blog: "https://medium.com/@muskangarg994/outreachy-101-d285507c54f7"
+        mentors:
+         - Florian Angeletti
+  - name: Summer 2021
+    projects:
+      - title: "Create opam package search"
+        description: >
+          opam is the source-based package manager for OCaml code. This project comprises of writing a new web
+          client for rendering output from the opam package database. There is a JSON endpoint on opam.ocaml.org
+          which provides information about packages which would provide metadata about the packages.  We can extend
+          this JSON metadata to include all the opam packages (not just the top 10) and use that to power a search
+          frontend for the website. This may include presenting the data as a GraphQL endpoint with the frontend
+          querying that endpoint using GraphQL.
+        mentee: "Odinaka Joy"
+        blog: "https://www.dinakajoy.com"
+        mentors:
+         - Sonja Heinze
+         - Patrick Ferris
+      - title: "Improve the ocaml.org website"
+        description: >
+          OCaml.org is the main website for OCaml, a functional, typed, high-level programming language. This project
+          revolves around improving the website on multiple different fronts including: layout, accessibility and content.
+        mentee: "Diksha Gupta"
+        blog: "https://dikshagupta99.wordpress.com/blog/"
+        mentors:
+         - Isabella Leandersson
+         - Patrick Ferris
+         - Gargi Sharma
+      - title: "Improve the ocaml.org website"
+        description: >
+          OCaml.org is the main website for OCaml, a functional, typed, high-level programming language. This project
+          revolves around improving the website on multiple different fronts including: layout, accessibility and content.
+        mentee: "Shreya kumari Gupta"
+        blog: "https://shreyaswikriti693235797.wordpress.com/blog/"
+        mentors:
+         - Isabella Leandersson
+         - Anil Madhavapeddy
+         - Patrick Ferris
+         - Gargi Sharma
+  - name: Winter 2021
+    projects:
+      - title: "Build a monitoring dashboard for OCaml.org"
+        description: ""
+        mentee: "Jiae Kam"
+        blog: "https://jiaek.wordpress.com"
+        mentors:
+         - Thibaut Mattio
+      - title: "Improve the OCaml meta-programming ecosystem"
+        description: ""
+        mentee: "Aya Sharaf"
+        blog: "https://outreachyxocaml.wordpress.com"
+        mentors:
+         - Shon Feder
+         - Sonja Heinze
```

Comment:
```suggestion
      - title: "Improve the OCaml meta-programming ecosystem"
        description: "It’s common for programming languages to provide some way to meta-program in order to preprocess code before reaching the last compilation step, for example, in the form of macros or templates. The OCaml compiler doesn't provide a full built-in macro system, but the OCaml parser does provide syntax meant for preprocessing purposes: attributes (https://v3.ocaml.org/docs/metaprogramming#attributes-and-derivers) and extension points (https://v3.ocaml.org/docs/metaprogramming#extension-nodes-and-extenders). We -the OCaml community- also have an official framework, called `ppxlib`, to write preprocessors -called PPXs- based on that syntax and integrate them into the compilation process.
        
        However, it's on the OCaml community to write and provide important PPXs to the OCaml developers. We've noticed that having the most important PPXs under the official PPX github organization -next to `ppxlib`- is helpful: developers can easily find them; developers can trust them; they're well-written and hygienic, so that developers can use them as how-to guides for writing other PPXs. In this project, you'll write (s)ome of those official standard PPXs."
        mentee: "Aya Sharaf"
        blog: "https://outreachyxocaml.wordpress.com"
        mentors:
         - Shon Feder
         - Sonja Heinze
```

Reply:
It seems like this is badly formatted. Could you let me know how to do a line-break in Ood?

Reply:
Thanks for the updates and welcome to wonderful world of line-breaks in YAML https://yaml-multiline.info (probably want a `|` and no need for the `"` or `\n`)

Reply:
>(s)ome of those official standard PPXs

are the parentheses intentional?